modules/gapi/test/gpu/gapi_operators_tests_gpu.cpp

```diff
@@ -58,17 +55,15 @@ INSTANTIATE_TEST_CASE_P(BitwiseOperatorTestGPU, MathOperatorMatScalarTest,
                                        cv::Size(640, 480),
                                        cv::Size(128, 128)),
                                 Values(-1),
-/*init output matrices or not*/ testing::Bool(),
                                 Values(CORE_GPU),
-                                Values(AbsExact().to_compare_f()),
+                                Values(AbsExact().to_compare_obj()),
```

Comment:
not sure if this rename is necessary, `to_compare_f` could work fine (where `f` is for functor).

Reply:
I don't want to break perf tests so this is the "compatibility" mode

Reply:
ok!